### DISCUSSION TITLE
Welcome to markdown-preview.nvim Discussions!

### DISCUSSION THREAD

**Discussion by iamcco** [iamcco/markdown-preview.nvim#271] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by keithkeekw** [comment]
Hi @iamcco, thanks for the wonderful plugins. I am wondering if there's any sort of roadmap for this plugin?

**Reply by iamcco** [reply]
Not have clear plan, when i have time I want to implement the features that request in the issue list, such as https://github.com/iamcco/markdown-preview.nvim/issues/58

**Reply by keithkeekw** [reply]
Yeah! I am following that issue as well. As for now, I am using .md file instead of .wiki for my VimWiki. Hopefully that can be implemented soon.
